app/main.py

```diff
@@ -12,11 +12,9 @@ def create_person_list(people: list) -> list:
         Person(person["name"], person["age"])
         for person in people
     ]
-    for person in people:
-        if person.get("wife"):
-            Person.people[person["name"]].wife =\
-                Person.people[person["wife"]]
-        elif person.get("husband"):
-            Person.people[person["name"]].husband =\
-                Person.people[person["husband"]]
+    for man in people:
```

Comment:
no, `man` is not a good variable name here. Better create intermediate variables wife/husband, get needed person and write it in this variables and then use them